src/main/java/ua/danit/final_project/entities/Schedule.java

```diff
@@ -7,23 +7,17 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'javax.persistence.Id'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=108&issues=AWXEVMDG_eEZPtUmhoyw&open=AWXEVMDG_eEZPtUmhoyw)

[ISSUE_KEY]: AWXEVMDG_eEZPtUmhoyw